Refactoring the Contribution Docs (#130)
moja-global/About_moja_global: Contributing/How-To-Manage-Backlog-System.md

## moja global Backlog system

The moja global Backlog system is designed to be a simple system to manage the backlog of bugs and features. Two GitHub tools are used to manage the backlog, **Issues** and **Projects**. 

> Issues are used for managing bugs associated with existing code or for requesting feature enhancements within existing repositories. To create a new bug to add to the backlog check [how to report bugs](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Report-Bugs.md). To create a new feature request check [how to create a new feature](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Request-a-New-Feature.md).

Author: Done!

> Projects are used where the feature (or a new functionality) is being proposed is best implemented as a new repository. An example might be a new FLINT module. For tips on setting up a new project check out how you can create [a new project](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Start-a-New-Project.md).

Reviewer: ```suggestion
Projects are used where the proposed feature (or a new functionality) is best implemented as a new repository. An example might be a new FLINT module. For tips on setting up a new project check out how you can create [a new project](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Start-a-New-Project.md).
```

Author: Done!

The backlog for any particular repository is managed by one or multiple maintainers. The role of the maintainer is to:

1.  Maintain the repository, including ensuring the list of issues is up to date and tracking the status of the code.
2.  Review and accept proposed changes to the repository they are responsible for.

> 3.  Add new issues for bugs and features for the repository where identified. Contributors are also advised to add issues for bugs and features.

Reviewer: ```suggestion
3.  Add new issues for bugs and features for the repository where identified. Contributors are also advised to add new issues for bugs and features if not already available.
```

Author: Done!

4.  Assign a priority to each issue using the issue labels: **Priority = High**, **Priority = Medium** and **Priority = Low**.

> 5.  If useful, coordinate with the maintainer during contributor meetings to prioritise issues and projects.

Reviewer: ```suggestion
5.  If useful, coordinate with other maintainers during contributor meetings to prioritise issues and projects.
```

Author: Done!

6.  Ensure that issues assigned to the contributors are being worked on.

Maintainers should set up the `watch` capacity for projects that they manage to ensure that they are aware of anything that is occurring in the project. It is also recommended that they `star` the relevant repositories of the project while they are working as the maintainer.

> If you have experience and want to help, please send an email to [tsc@moja.global](mailto:tsc@moja.global).

Author: Done!
